src/AutoRest.CSharp/Mgmt/AutoRest/MgmtOutputLibrary.cs

```diff
@@ -926,8 +929,19 @@ private TypeProvider BuildResourceData(InputType inputType, MgmtObjectType? defa
         private Dictionary<string, OperationSet> CategorizeOperationGroups()
         {
             var rawRequestPathToOperationSets = new Dictionary<string, OperationSet>();
-            foreach (var inputClient in _input.Clients)
+            foreach (var inputClient in _inputClients)
             {
+                // If the client has no operations, we skip it
+                if (inputClient.Operations.Count == 0)
+                {
+                    continue;
+                }
+
+                // Skip "Operations" operation group in the Operations client
+                if (inputClient.Name == "Operations")
+                {
+                    continue;
+                }
```

Comment:
here we need a TODO item as well because this is definitely not a smart way to remove the `Operations_List` method.

If possible I think we should remove this here, and add the internal or remove to those operations in tsp.
For swagger input, we should not include them, ie, remove them from the schema first, then convert, instead of convert then remove.

Reply:
That has been done, this should be removed.

Reply:
This has been removed.